crowdin.yml

```diff
@@ -0,0 +1,22 @@
+#
+# Juicebox Crowdin credentials
+#
+'project_id': '492549'
+'api_token': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiJ3UUVxdmhVM3ZMT2EyWGljbVV5VCIsImp0aSI6ImE1ODFlMTZlMjRmYzAwOWVlMzgzYjVlYTE4Mzg3YjlhNmQ5MjVjZmVkNzg3M2UyMmU4ZDIxOGM4MDI0ZDMwOWJlMzY3ZmUyN2VmZjc3YmUxIiwiaWF0IjoxNjQxMTc4NDE0LCJuYmYiOjE2NDExNzg0MTQsImV4cCI6MTY0Mzc3MDQxNCwic3ViIjoiMTUwODc4NDciLCJkb21haW4iOm51bGwsInNjb3BlcyI6WyJwcm9qZWN0Il0sImFzc29jaWF0aW9ucyI6WyIqIl0sInNlc3Npb24iOjB9.hJ-PPDgbqGqEQnA4RKR1PaLaSZJ-vYYwCkJvvi4vdaxRJTjHgG7Y3u6vA32TH3fUDtWaLpqr52HYMdVSgvDvFOK_wQwqxurPqJ4SPhntWjHJfoF2d22CFcIMn7HIzYVD7cqWLXUGlgHQggiHsU-WKU42PzVGst-7nKtvlSYjPVZj_-XsZUbH0iHYF7AutXZcY1P0eeD2z6dLPhl252J5GYTwMXuWSAZjNeiZq3RtZkrflmgyWOlva_aoGGy15A6iZI2VXs7qhvkq4FebXrYBrUMWO3x_gwg_Oi2rN6Q8Ay_tw7ZtqWMjk_LYIweQtX6U9d_tvGO1Fr6N9xDe8MErAp_pSWICLpHpTpMujWo5UUXZfJq_HL8N1tU9ycTa-MOOQVKBI8zissCVB0eem7f4izfb2_D14xD90l-sd_uoTEyyyksEE3Hl3-kcD49nykkfZp0XucaafDxDkWmKLtn9XcCk4Sf2juPYrCv5nd--md0JblYa1gduhvo98lGb1gmdKEF1SHrPD5Joalm48WP9A0dvNCkQ6jRKV5Qp27iZakGoUL8UNNRC_-V0o6uOtiwr4tu-P_c_saCPisep71JHfj8keoBSMsrzD2ktP2DmJheouEjan7GGmBOMhJNfob-VN0jbckk5avBbpuZyKBCUHmRuhihekarfEy5i_765ABw'
```

Comment:
**issue**: lets not expose this 😛 

Regardless, we should have a single source of truth for config, currently it's this file and the github action files.

Reply:
Spent 2 very frustrating days trying to get the Github secret vars to work, only to finally figure out they don't work on forked repos.... 

It was trying to read from the upstream repo where the secrets don't exist. Confident this will work when we put it on main in the base repo (can't test on a feature branch because it won't be able to read the secrets)

Reply:
@johnnyd-eth feature branches should be able to read Github secrets, have you tried making this a feature branch instead of a fork?

(also in general, using feature branches makes it easier for other folks to pull down and review code)

Reply:
@johnnyd-eth I think additions to GitHub Actions should still run in branches in the main repo - give it a go 🙏

Reply:
> @johnnyd-eth feature branches should be able to read Github secrets, have you tried making this a feature branch instead of a fork?
> 
> (also in general, using feature branches makes it easier for other folks to pull down and review code)

@peripheralist Oh yep ok sounds good. When you get a chance could you please add CROWDIN_PERSONAL_TOKEN to the Github secrets with the value:

eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiJ3UUVxdmhVM3ZMT2EyWGljbVV5VCIsImp0aSI6ImE1ODFlMTZlMjRmYzAwOWVlMzgzYjVlYTE4Mzg3YjlhNmQ5MjVjZmVkNzg3M2UyMmU4ZDIxOGM4MDI0ZDMwOWJlMzY3ZmUyN2VmZjc3YmUxIiwiaWF0IjoxNjQxMTc4NDE0LCJuYmYiOjE2NDExNzg0MTQsImV4cCI6MTY0Mzc3MDQxNCwic3ViIjoiMTUwODc4NDciLCJkb21haW4iOm51bGwsInNjb3BlcyI6WyJwcm9qZWN0Il0sImFzc29jaWF0aW9ucyI6WyIqIl0sInNlc3Npb24iOjB9.hJ-PPDgbqGqEQnA4RKR1PaLaSZJ-vYYwCkJvvi4vdaxRJTjHgG7Y3u6vA32TH3fUDtWaLpqr52HYMdVSgvDvFOK_wQwqxurPqJ4SPhntWjHJfoF2d22CFcIMn7HIzYVD7cqWLXUGlgHQggiHsU-WKU42PzVGst-7nKtvlSYjPVZj_-XsZUbH0iHYF7AutXZcY1P0eeD2z6dLPhl252J5GYTwMXuWSAZjNeiZq3RtZkrflmgyWOlva_aoGGy15A6iZI2VXs7qhvkq4FebXrYBrUMWO3x_gwg_Oi2rN6Q8Ay_tw7ZtqWMjk_LYIweQtX6U9d_tvGO1Fr6N9xDe8MErAp_pSWICLpHpTpMujWo5UUXZfJq_HL8N1tU9ycTa-MOOQVKBI8zissCVB0eem7f4izfb2_D14xD90l-sd_uoTEyyyksEE3Hl3-kcD49nykkfZp0XucaafDxDkWmKLtn9XcCk4Sf2juPYrCv5nd--md0JblYa1gduhvo98lGb1gmdKEF1SHrPD5Joalm48WP9A0dvNCkQ6jRKV5Qp27iZakGoUL8UNNRC_-V0o6uOtiwr4tu-P_c_saCPisep71JHfj8keoBSMsrzD2ktP2DmJheouEjan7GGmBOMhJNfob-VN0jbckk5avBbpuZyKBCUHmRuhihekarfEy5i_765ABw

Reply:
let's not put secrets in public chat threads 😄

i'll set this up

Reply:
My bad! 😅